[FR] Collab Information (#2423)
ppy/osu-wiki: wiki/Guides/Collab_Information/fr.md

# Informations sur les collaborations

Les collaborations sont une sorte de mapping difficile à réaliser. Il y a beaucoup de facteurs à inclure.... Voici quelques conseils à prendre en compte lors d'une collaboration.

## Être sûr

Qu'est ce que cela veut dire ?

-   Être sûr de quelle **chanson** vous choississez.
-   Être sûr de séparer les **parties et envisager les pauses**. (C'est beaucoup plus amusant si vous pouvez diviser les parties de manière uniforme.) 
-   Être sûr que tout le monde est d'accord sur les **options de la map**.

> -   Être sûr de demander aux **mappers** avant de leur demander une map. (C'est plus poli.)

Reviewer: J'avoue que je ne comprends pas ce qu'ils veulent dire à la base, mais je ne pense pas que cette traduction soit correcte.

Je ne suis pas certain, mais je dirais:
```suggestion
-   Être sûr de demander à tous les **mappers** que vous souhaitez inviter avant de les inclure sur une map. (C'est plus poli.)
```

-   Être sûr que vous pouvez **suivre jusqu'au bout** la collaboration. (Que vous puissiez continuer et que tout le monde dans la collaboration soit présent et attentif à tout moment)

> Cette étape est, en essence, une étape de confirmation pour voir si faire une collaboration est fait pour vous. A partir d'ici, il y a quelques points à suivre. 

Reviewer: ```suggestion
Cette étape sert, en quelque sorte, de confirmation pour voir si les collaborations sont faites pour vous. À partir d'ici, il y a quelques points à suivre.
```

## Sélection de la chanson

Une étape qui peut être facilement écartée, mais dangereux si vous faites comme cela.

-   Choississez une chanson que tout le monde accepte.
-   Le timing est toujours important.
-   Les chansons qui ont des modifications distinctes dans la musique ou des parties distinctes sont généralement celles à choisir lors d'une collaboration.

## Partitionnement

Généralement il est préférable de savoir de quelle partie vous vous occupez et ce que les autres personnes feront avant de commencer à faire le mapping.

-   Premièrement, il est recommandé de bien marquer votre section de la chanson. Faites le autant que possible.

> -   En ne prenant pas en compte qui commence a faire le mapping, il est plus facile de commencer du début à la fin plutôt que faire le mapping de toutes vos parties. De cette façon, il est plus facile de mieux jouer et mieux circuler.

Reviewer: ```suggestion
-   Peu importe qui commence, il est bien plus facile de mapper du début à la fin plutôt que chacun de son côté. De cette façon, il est plus facile de s'organiser et la map se jouera mieux.
```

Je ne suis pas sûr de la seconde phrase de ma suggestion, mais je ne vois pas le sens de `circuler` dans ce contexte.

> -   Souvenez vous de partitionner les pauses. **La plupart des personnes oublient cela.**

Reviewer: ```suggestion
-   Souvenez vous de séparer les parties avec des pauses. **La plupart des personnes oublient cela.**
```

> ## Options des map

Reviewer: ```suggestion
## Options de la map
```

> Si vous ne pouvez pas vous mettre d'accord sur les options des maps...cela va être préjudiciable à la map.

Reviewer: ```suggestion
Si vous ne pouvez pas vous mettre d'accord sur les options de la map, sa qualité risque d'en prendre un coup. Vous devriez vous mettre d'accord sur :
```

On reste sur une seule map pour ce guide ^^
Je me suis permis de reformuler cette section, à voir si ça te va.

> -   Convenez une vitesse de slider. C'était un grand tueur pour Last Time Travel et Gonna Be Here.

Reviewer: ```suggestion
-   la vitesse des sliders. C'était un gros problème pour Last Time Travel et Gonna Be Here.
```

> -   Convenez de la distance et de l'espacement en général. Regardez [Be Your Wings](https://osu.ppy.sh/beatmapsets/8518).

Reviewer: ```suggestion
-   la distance et l'espacement des notes. Regardez [Be Your Wings](https://osu.ppy.sh/beatmapsets/8518).
```

Pas sûr que ma suggestion soit plus correcte.

> -   Convenez des options de difficulté.

Reviewer: ```suggestion
-   la configuration de la difficulté.
```

## Mappers

> C'est la partie qui rend le plus de personnes folles. Avoir les mappers de vos rêves n'est pas facile et si vous réussissez à l'avoir, vous devez être prêt sur tous les aspects avant de commencer.

Reviewer: ```suggestion
C'est la partie qui rend le plus de personnes folles. Avoir les mappers de vos rêves n'est pas facile, mais si vous réussissez à les avoir, vous devez être prêt sur tous les aspects avant de commencer.
```

-   Développez vos propres compétences en mapping. C'est un des aspects clés. Si la personne avec laquelle vous collaborez n'aime pas comment vous mappez... Vous pouvez dire au revoir à votre collaboration.

> -   Choississez quelqu'un qui à un style similaire au votre. Il sera plus facile d'être sur la même longueur d'onde quand vous ferez le mapping.

Reviewer: ```suggestion
-   Choisissez quelqu'un qui a un style similaire au vôtre. Il sera plus facile d'être sur la même longueur d'onde quand vous ferez le mapping.
```

-   Soyez sûr que vous pouvez vous contacter facilement... **C'est important car si les personnes avec qui vous travaillez disparaissent pour n'importe quelle raison, vous serez coincés.**

Souvenez-vous qu'une bonne map est une map qui prend du temps à faire et qui a un grand potentiel. Les collaborations sont souvent très difficile à faire ranked. Quelques raisons à cela :

-   Vous avez plus d'un mapper, et si les modders trouvent un problème dans une section d'un autre mapper, l'autre mapper devra le résoudre.

> -   Certaines parties peuvent entrer en conflit et le flux de la map peut être très difficile à obtenir.

Reviewer: On pourrait garder `flow` au lieu de le traduire. Je pense que la plupart des gens comprennent le terme et je trouve que c'est difficile d'avoir un équivalent en français.

On aurait quelque chose du genre `Certaines parties peuvent entrer en conflit et le flow de la map peut être difficile à prendre en main`
